.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea/
+/build/
+/.gradle/
+/src/main/java/SimpleTestMain.java
```

Comment:
Why do you need to ignore this file?

Reply:
This file is ignore /src/main/java/SimpleTestMain.java, because is only for fast local test